eth/gasprice/gasprice.go

```diff
@@ -32,12 +32,14 @@ import (
 const sampleNumber = 3 // Number of transactions sampled in a block
 
 var DefaultMaxPrice = big.NewInt(500 * params.GWei)
+var DefaultIgnorePrice = big.NewInt(0 * params.GWei)
```

Comment:
If you set the default to `2 * params.Wei`, then we'd keep the current (just merged) behaviour

Reply:
Thanks, I rebased off master and applied this change as well as your other suggestions as part of the updated 4096b9aba790b850ad495e560bb56557aaeae31f commit.